pkg/artifactory/resource_artifactory_ldap_group_setting_test.go

```diff
@@ -0,0 +1,97 @@
+package artifactory
+
+import (
+	"encoding/xml"
+	"fmt"
+	"github.com/go-resty/resty/v2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
```

Comment:
Please re-organize these imports by alpha, and groups

Reply:
ran gofmt. Looks sorted and grouped ok. Please let me know otherwise.